.github/workflows/commit-queue.yml

```
@@ -0,0 +1,76 @@
+---
+# This action requires the following secrets to be set on the repository:
+#   GH_USER_NAME: GitHub user whose Jenkins and GitHub token are defined below
+#   GH_USER_TOKEN: GitHub user token, to be used by ncu and to push changes
+#   JENKINS_TOKEN: Jenkins token, to be used to check CI status
+
+name: Commit Queue
+
+on:
+  schedule:
+    - cron: "*/5 * * * *"
+
+jobs:
+  commitQueue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # A personal token is required because pushing with GITHUB_TOKEN will
+          # prevent commits from running CI after they land on mater. It needs
```

```suggestion
          # prevent commits from running CI after they land on master. It needs
```
Although I like the concept of a "mother branch." :-)